client/src/components/molecules/Choice.tsx

```diff
@@ -0,0 +1,67 @@
+import { theme } from "@/theme";
+import styled from "@emotion/styled";
+import type { Choice } from "biseo-interface/agenda";
+import { Box, Text } from "@/components/atoms";
+import { SelectIcon } from "@/assets";
+
+const ChoiceContainer = styled.div<{
+  chosen?: boolean;
+}>`
+  border-radius: 5px;
+  background-color: ${props =>
+    props.chosen ? props.theme.colors.blue600 : props.theme.colors.white};
+  border: 1px solid ${props => props.theme.colors.gray200};
+  padding: 6px 13px 6px 13px;
+  width: 340px;
+  height: 30px;
+  gap: 10px;
+`;
+
+type ChoiceProps = {
+  choice: Choice;
+  chosen: boolean;
+  onClick: () => void;
+};
+
+const ChoiceComponent: React.FC<ChoiceProps> = ({
+  choice,
+  chosen,
+  onClick,
+}) => {
+  const iconColor = chosen ? theme.colors.blue600 : theme.colors.gray500;
+  const textColor = chosen ? "white" : "gray500";
+  return (
+    <ChoiceContainer onClick={onClick} chosen={chosen}>
+      <Box dir="row" gap={10}>
+        <SelectIcon stroke={iconColor} />
+        <Text variant="body" color={textColor}>
+          {choice.name}
+        </Text>
+      </Box>
+    </ChoiceContainer>
+  );
+};
```

Comment:
`iconColor`랑 `textColor`가 어차피 한 번씩만 쓰이는 것 같아서 따로 변수로 만들지 말고 이런 식으로 pure component로 바꾸면 더 좋을 것 같아요!!

```suggestion
const ChoiceComponent: React.FC<ChoiceProps> = ({
  choice,
  chosen,
  onClick,
}) => (
  <ChoiceContainer onClick={onClick} chosen={chosen}>
    <Box dir="row" gap={10}>
      <SelectIcon stroke={chosen ? theme.colors.blue600 : theme.colors.gray500} />
      <Text variant="body" color={chosen ? "white" : "gray500"}>
        {choice.name}
      </Text>
    </Box>
  </ChoiceContainer>
);
```